### DISCUSSION TITLE
Suggestions to improve helloworld example

### DISCUSSION THREAD

**Discussion by gftea** [aptos-labs/aptos-developer-discussions#233] [Dev Doc and Tutorial Feedback]
I suggest to have helloworld to demonstrate more aptos capability which give user a better helicopter view.
The example includes module event and view functions, but the tutorial does not explain how to query. suggest to add below
# Suggestions
1. Add example query for module event
2. Add example view call to view the message

# Extra
If move has similar tool as `cargo doc` then can generate documentation? If it is available, it would be better to write docs in the codes, and generate the docs, which provide seamless learning experience.

**Comment by kshitijc1506** [comment]
Hi, thanks for the feedback! I'll look into this in the coming 1-2 weeks.
